brave/browser-laptop
# Add preparation for upcoming Ubuntu 18.10 Cosmic Cuttlefish and other cleanups of the apt install

This PR removes all the old Ubuntu releases that are unmaintained and don't get any security updates, adds support for Ubuntu Cosmic and organizes the list a bit better. Brave had a rocky start when 18.04 was released so this should hopefully make Brave work on the new version from day one! 

## Submitter Checklist:


## Test Plan:

I can't test this since it requires Brave to be built for Apt, but please give it a shot if you can 👍
I am willing to spin up a WM and test installing this on Cosmic CUttlefish when merged...

1. Have a channel ready for build (ex: beta)
2. Connect to our signator box and unlock the GPG key
3. Login to Jenkins and run the `browser-laptop-release-linux` job
  - if we wanted to test before accepting this PR, we can modify that job to checkout from this branch
4. After it finishes, have an Ubuntu 18.10 [Daily Build](http://cdimage.ubuntu.com/daily-live/current/HEADER.html) VM ready 
5. [Follow instructions here](https://github.com/brave/browser-laptop/blob/master/docs/linuxInstall.md)
6. Brave should install
